Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,27 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
+            
             // To do: call your implementation. 
```

Comment:
The comment can be removed, because you already call your implementation.

Reply:
I have removed all the comments as suggested.